Arashayo/TabletopHaven
# Add friends view <all previous branches combined>

## Related Issue
https://app.clickup.com/t/86bxew0u5

## Changes list
Added friend list and user search
Refactored styling across whole app
Combined all branches to one to fix all styling issues across the app
Unified the styling so it works the same in every view

## Testing instructions
Run both backend and frontend
Go to Friends
Try searching users by name
Check if friends list is displayed correctly

## Screenshots (if appropriate):
